src/Umbraco.Forms.Integrations.Crm.Hubspot/umbraco-marketplace.json

```diff
@@ -0,0 +1,31 @@
+{
+  "AuthorDetails": {
+    "Description": "Umbraco HQ",
+    "Url": "https://umbraco.com/",
+    "ImageUrl": "https://avatars.githubusercontent.com/u/1419552?s=200&v=4"
+  },
+  "Category": "Forms Extensions",
+  "LicenseType": "Free",
+  "PackageType": "Package",
+  "PackagesByAuthor": [ "UmbracoFormsIntegrationsAutomationZapier", "UmbracoFormsIntegrationsCommerceemerchantpay", "UmbracoFormsIntegrationsCrmActiveCampaign" ],
```

Comment:
These package Ids should be the NuGet package Ids, e.g. `Umbraco.Forms.Integrations.Automation.Zapier`.